src/silx/gui/widgets/_SliderWithSpinBox.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from silx.gui import qt
+from silx.gui.utils import blockSignals
+
+
+class SliderWithSpinBox(qt.QWidget):
```

Comment:
Since this widget is only used in `ColormapDialog`, let's move this module to `silx.gui.dialog`, there is no reason to have it in `silx.gui.widgets`.